apps/recnet/src/app/[handle]/Profile.tsx

```diff
@@ -2,27 +2,24 @@
 
 import { zodResolver } from "@hookform/resolvers/zod";
```

Comment:
[nit] why did you include `[]` to the directory name here?

Reply:
It marks this is a dynamic route: https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes

The request to `/yoav`, `/swh00tw`, `joanne`, ... will be direct to this route